docs/tutorials/tensorflow_inference_and_training.md

```diff
@@ -83,8 +83,9 @@ for batch in [2, 2, 4, 1, 1, 8, 8, 2, 16, 2]:
 
 The above code shows the complete process to optimize BERT inference model with
 BladeDISC. Please refer to [TensorFlow BERT Inference
-Example](/examples/TensorFlow/Inference/BERT) for more scripts to compare the
-performance of BladeDISC optimization with naive TensorFlow and XLA.
+Example](https://github.com/alibaba/BladeDISC/tree/main/examples/TensorFlow/Inference/BERT)
```

Comment:
My personal recommendation is that by keeping the folder path, which helps reference each tagged release or commit, the URL would always reference the latest commit.

Reply:
The folder path will bring 404 error in the exported htmls. Here we should guide the reader to somewhere the example files can be seen.